src/atom_finder/classifier/operator-precedence.clj

```diff
@@ -74,48 +81,40 @@
   "returns a collection of operator group pairs between the node and its children
   , if a second parameter is passed, use that as the collection instead of the children"
   ([node]
-   (->> node
-        children
-        (map operator-group)
-        (remove nil?)
-        (map (fn [child-group] (sort [(operator-group node) child-group])))))
+   (group-pair node (children node)))
 
   ([node collection]
    (->> collection
         (map operator-group)
         (remove nil?)
         (map (fn [child-group] (sort [(operator-group node) child-group]))))))
 
-
-;;
-;;The following 3 functions are for the binary operator special case
-;;
-(defn rvalue-unary-operator?
+;;====================================
+;;For binary operator special case
```

Comment:
do you really mean binary, not unary here?? if so, please explain more, I don't understand